docs/articles/nunit/getting-started/installation.md

```diff
@@ -1,43 +1,149 @@
+
+```csproj
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+        <PackageReference Include="NUnit" Version="3.13.3" />
+        <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
+        <PackageReference Include="NUnit.Analyzers" Version="3.7.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="coverlet.collector" Version="6.0.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+     </ItemGroup>
+
+</Project>
+
+```
+
+It will create a `UnitTest1.cs` file with the following contents:
+
+```cs
+namespace TestProject1;
+
+public class Tests
+{
+    [SetUp]
+    public void Setup()
+    {
+    }
+
+    [Test]
+    public void Test1()
+    {
+        Assert.Pass();
+    }
+}
+```
+
+and a `Usings.cs` file:
+
+```cs
+global using NUnit.Framework;
+```
+
+## Adding NUnit to an existing test project
+
+If you need to update an existing test project to use NUnit, it may a standard class lib you want to convert to a test project, or you want to add NUnit to a project that already has XUnit or MSTest, you can do that by adding the itemgroup in the csproj above to your project file, and then the appropriate `using NUnit.Framework;` in your code.
+
+Also note that multiple different test frameworks (XUnit, MSTest, NUnit) can co-exist in the same test project, preferably in different files to avoid name clashes, see [this old blog post](https://devblogs.microsoft.com/devops/part-2using-traits-with-different-test-frameworks-in-the-unit-test-explorer/) for details.
+
+## TL;DR
+
+All of the above ways to create a new NUnit project will result in the same structure, the same test class and methods, and the same packages being installed. That means you can do it any way you like, you can start with one tool and continue with another.
+
+The template used is maintained as part of dotnet and will be upgraded when dotnet updates. All the above pick the template from dotnet, which is why the end result is the same.
+
+## I want to do it the old and hard way
+
+This is from the older documentation, and is still valid, but we recommend you use the above methods.
 
 1. Full NUnit install via NuGet.
 2. NUnitLite install via NuGet.
 3. Zip and/or MSI file download.
 4. Combined Approach
 
-## Using NuGet Packages
+### Using NuGet Packages and the NUnit Console
```

Comment:
❓ Are these intended to be child items of `I want to do it the old and hard way`? Otherwise they probably should stay as H2s.